tests/docs/test_docs.py

```diff
@@ -22,10 +22,11 @@
     not CHEM_INSTALLED, reason="Optional chem dependency not installed."
 )
 @pytest.mark.parametrize("file", doc_files, ids=doc_files)
-def test_code_executability(file: Path):
+def test_code_executability(file: Path, campaign):
```

Comment:
Type hint missing

Reply:
nearly all tests do not hint types of fixtures

Reply:
Yes, but it is weird if one `arg` has a type hint but the other doesn't. I would prefer at least consistency within signatures, but feel free to resolve if you disagree.